src/app/view/view-controls/view-controls.component.html

```diff
@@ -1,28 +1,24 @@
-<form class="form-inline d-flex">
-  <span i18n="@@view.perspective" class="col-form-label font-weight-bold">Perspective</span>:
-  <div class="form-control dropdown mx-3 p-0">
-    <div class="dropdown-toggle py-1 px-2" id="dropdownMenuButton" data-toggle="dropdown"
-         aria-haspopup="true" aria-expanded="false">
+<form class="form-inline d-flex flex-nowrap">
+  <div class="form-control dropdown mr-2 p-0" title="Perspective" i18n-title="@@view.perspective">
+    <div class="dropdown-toggle py-1 px-2 d-flex flex-nowrap align-items-center"
+         id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
       <i class="fa-fw {{getIconForPerspective(view.perspective)}}"></i>
-      <span class="ml-1">{view.perspective, select, postit {Post-it} search {Search} table {Table}}</span>
+      <span class="ml-1" i18n="@@view.perspective">{view.perspective, select, postit {Post-it} search {Search} table {Table}}</span>
```

Comment:
Cannot we use non-breaking hyphen here (`&#8209;`) for Post-it name? Because at the moment, this name is split into two lines on very small screens which is not really what we want.

Reply:
Of course...

Reply:
Maybe `white-space: nowrap` is better, isn't it?